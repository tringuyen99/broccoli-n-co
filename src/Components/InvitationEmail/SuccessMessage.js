//Components
import Container from 'react-bootstrap/Container';
import Button from 'react-bootstrap/Button';

function SuccessMessage(props) {
    return (
        <Container>
            <h1 className="display-6 fw-bold">All done!</h1>
            <p className="fw-light">Broccoli&Co</p>
            <br/>
            <p className="fw-normal-5 ">
                Our team will reach you with a confirmation email along with the details about the invitation.
                Looking forward to seeing you there!
            </p>
            <div className="d-grid gap-2 mt-5">
                <Button variant="primary" size="lg" onClick={props.onClose}>
                    OK!
                </Button>
            </div>
        </Container>
    )
}

export default SuccessMessage;