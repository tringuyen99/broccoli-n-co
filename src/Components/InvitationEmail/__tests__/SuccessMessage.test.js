import React from 'react';
import renderer from 'react-test-renderer';
import SuccessMessage from '../SuccessMessage';

const propsStub = {
    onSuccess: () => {},
};

it('SuccessMessage renders correctly', () => {
    const tree = renderer
        .create(<SuccessMessage {...propsStub} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});