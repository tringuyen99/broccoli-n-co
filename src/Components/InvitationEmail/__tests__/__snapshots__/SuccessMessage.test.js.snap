// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="display-6 fw-bold"
  >
    All done!
  </h1>
  <p
    className="fw-light"
  >
    Broccoli&Co
  </p>
  <br />
  <p
    className="fw-normal-5 "
  >
    Our team will reach you with a confirmation email along with the details about the invitation. Looking forward to seeing you there!
  </p>
  <div
    className="d-grid gap-2 mt-5"
  >
    <button
      className="btn btn-primary btn-lg"
      disabled={false}
      type="button"
    >
      OK!
    </button>
  </div>
</div>
`;
